src/modules/block.js

```diff
@@ -13,17 +13,20 @@ module.exports = bot => {
       msg.channel.createMessage(`Blocked <@${userId}> (id ${userId}) from modmail`);
     }
 
-    let logText = `**Blocked:** ${thread.user_name} (${thread.user_id}) was blocked.`;
+    let logText = `**Blocked:** `;
 
     if (! thread && args.length > 0) {
       // User mention/id as argument
-      const userId = utils.getUserMention(args.join(' '));
+      const userId = utils.getUserMention(args[0]);
```

Comment:
`args.shift()` would be preferable, this removes the need to slice later on

Reply:
Didn't even notice this - Sounds good